### ISSUE TITLE
Could not find required ModuleFederation plugin options in the webpack config.

### ISSUE THREAD

**Issue by VatsalDave2102** [cloudbeds/webpack-module-federation-types-plugin#41]
Im trying to build a type safe MFE architecture, I have build two apps `container` and `remote`. While building types for `Button.tsx` in `container` app using `npx make-federated-types`. Im getting this error:

 `Could not find required ModuleFederation plugin options in the webpack config.`

Below is my code:

`container/federation.config.json`:
```
{
 "name": "container",
 "exposes": { "./Button": "./src/components/Button.tsx" }
}
```

`container/webpack.config.json`:
```
const HtmlWebPackPlugin = require("html-webpack-plugin");
const ModuleFederationPlugin = require("webpack/lib/container/ModuleFederationPlugin");
const federationConfig = require("./federation.config.json");

// define the package.json dependencies
const deps = require("./package.json").dependencies;

const configs = {
	appName: "container",
	appFileName: "remoteEntry.js",
	development: {
		PUBLIC_PATH: "http://localhost:8080/",
		REMOTE_PATH: "remote@http://localhost:8081/remoteEntry.js",
		PORT: 8080,
	},
	production: {
		PUBLIC_PATH: "http://localhost:8080",
		REMOTE_PATH: "remote@http://localhost:8081/remoteEntry.js",
		PORT: 8080,
	},
};

module.exports = (_, argv) => ({
	output: {
		// output path for the build files
		publicPath: configs["development"].PUBLIC_PATH,
	},

	resolve: {
		// the extensions that should be used to resolve modules
		extensions: [".tsx", ".ts", ".jsx", ".js", ".json"],
	},

	// the configuration for the dev server
	devServer: {
		hot: true, // enable webpack's hot module replacement feature,
		port: configs["development"].PORT,
		historyApiFallback: true,
		allowedHosts: "all",
		headers: {
			"Access-Control-Allow-Origin": "*",
			"Access-Control-Allow-Headers":
				"Origin, X-Requested-With, Content-Type, Accept",
		},
	},

	module: {
		rules: [
			{
				test: /\.m?js/,
				type: "javascript/auto",
				resolve: {
					fullySpecified: false,
				},
			},
			{
				test: /\.(css|s[ac]ss)$/i,
				use: ["style-loader", "css-loader", "postcss-loader"],
			},
			{
				test: /\.(ts|tsx|js|jsx)$/,
				exclude: /node_modules/,
				use: {
					loader: "babel-loader",
				},
			},
		],
	},

	// plugins configuration
	plugins: [
		new ModuleFederationPlugin({
			...federationConfig,
			name: configs.appName,
			filename: configs.appFileName,
			remotes: {
				remote: configs["development"].REMOTE_PATH,
			},
			exposes: {
				"./Button": "./src/components/Button.tsx",
			},
			shared: {
				...deps,
				react: {
					singleton: true,
					requiredVersion: deps.react,
				},
				"react-dom": {
					singleton: true,
					requiredVersion: deps["react-dom"],
				},
			},
		}),
		new HtmlWebPackPlugin({
			template: "./src/index.html",
		}),
	],
});

```

Am I missing something here? Or is there any bug that's causing it.

**Comment by steven-pribilinskiy** [comment]
[Here is the relevant piece of code](https://github.com/cloudbeds/webpack-module-federation-types-plugin/blob/2dfe60c71c94a9977028f5a8af660450abd6588f/src/bin/helpers/getOptionsFromWebpackConfig.ts#L25-L46). 

It should not complain, I see you've added `ModuleFederationPlugin` to the config

I would suggest to debug it (e.g. console.log the config, then if still not clear, put some console.log-s in the compiled js of the plugin)

**Comment by steven-pribilinskiy** [comment]
Resolved in 2.0.7
